package.json

```diff
@@ -26,36 +26,36 @@
     "@babel/preset-typescript": "^7.14.5",
     "@commitlint/cli": "^12.1.4",
     "@rollup/plugin-node-resolve": "^13.0.0",
-    "@rollup/plugin-replace": "^2.4.1",
-    "@types/acorn": "^4.0.5",
-    "@types/babel__core": "^7.1.14",
-    "@types/jest": "^26.0.23",
-    "@types/node": "^15.12.4",
-    "@typescript-eslint/eslint-plugin": "^4.28.0",
-    "@typescript-eslint/parser": "^4.28.0",
+    "@rollup/plugin-replace": "^2.4.2",
+    "@types/acorn": "^4.0.6",
+    "@types/babel__core": "^7.1.15",
+    "@types/jest": "^26.0.24",
+    "@types/node": "^16.0.1",
+    "@typescript-eslint/eslint-plugin": "^4.28.2",
+    "@typescript-eslint/parser": "^4.28.2",
     "babel-plugin-tester": "^10.1.0",
     "es5-proxy-compat": "^0.22.2",
-    "eslint": "^7.29.0",
+    "eslint": "^7.30.0",
     "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-jest": "^24.3.6",
     "eslint-plugin-lwc-internal": "link:./scripts/eslint-plugin",
     "execa": "^5.1.1",
     "fs-extra": "^10.0.0",
     "glob": "^7.1.7",
-    "husky": "^6.0.0",
+    "husky": "^7.0.1",
```

Comment:
From [Husky 7.x](https://github.com/typicode/husky/releases/tag/v7.0.0): 

> Improve .husky/ directory structure. .husky/.gitignore is now unnecessary and can be removed.

should we remove [ours](https://github.com/salesforce/lwc/blob/master/.husky/.gitignore)?

Reply:
Good catch! I didn't notice; I was just looking for breaking changes. 😅